tests/console/yast2_dns_server.pm

```diff
@@ -8,7 +8,7 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: New test for yast2-dns-server (service management)
+# Summary: Ensure that all combinations of running/stopped and active/inactive can be set for yast2 dns-server
```

Comment:
I guess with this change also the existing test module maintainer mgriessmeier would be happy if you take maintainership :)

Reply:
Sure, no problem ;)